blog/2022-12-6-benefits-challenges-localization.md

```diff
@@ -0,0 +1,87 @@
+
+* English to German 🇩🇪 text expands by 10-35%.
+* English to Finnish 🇫🇮 text shrinks by 25-40%.
+* English to Spanish 🇪🇸 text expands by 20-25%.
+
+To make sure all the text fits correctly to your UI, you should check how the translations look directly in your app. Tolgee makes this easier with the [Tolgee tools plugin](https://chrome.google.com/webstore/detail/tolgee-tools/hacnbapajkkfohnonhbmegojnddagfnj). You can translate your web application more effectively with the Tolgee plugin, which is an integral part of Tolgee ecosystem. With the Tolgee Tools plugin, you can also modify translations live on your page to see the location of all translations. This way, you will find out how exactly the page will look with the newly added translation and how the translation expanded or shrank.
+
+![translation expansion](/img/blog/translation_lenght.png)
+To tackle this issue with translated text expansion or shortage already in the designing phase of your app, we plan to release Tolgee Figma plugin soon. You’ll be able to make your design localization-ready right from the start and make collaboration between designers, translators, and developers way more efficient. 
+
+## 2. Right-to-left languages specifics
+
+Another challenge may arise while localizing right-to-left languages such as Arabic, Hebrew, or Persian. The different text flow can affect page layouts, user experience, and other aspects of your app. You have to make sure that those languages are correctly set to display. 
+
+Tolgee Chrome plugin may help you in the translation review phase to see how the final localized app will look like. Tolgee currently supports right-to-left (RTL) languages; however, we plan to make it even easier with a visual editor in Q1/2023. It will do the work with HTML characters and special characters in RTL languages a piece of cake. 
+
+
+## 3. Accurate translation
+
+Another challenge you may face when localizing your software is ensuring that all text is accurately translated. The best outcome you would certainly get is by hiring a professional translator. However, not all people want to make such an investment and prefer machine translations. However, machine translations are currently not at the stage when they can translate software solely on their own with perfect results. If you use machine translation and even if you choose [suitable MT provider](/blog/software-localization-machine-translation), you better ask a translator or native speaker to review the translations. You can learn how Tolgee can help you with MT translation in this [article.](/blog/machine-translation-in-tolgee)
```

Comment:
However, not all people want to make such an investment and prefer machine translations. However, machine translations are currently not at the stage when they can translate software solely on

-> To much however.